woof-code/rootfs-skeleton/usr/bin/xwin

```diff
@@ -31,11 +31,14 @@ fi
 
 if [ -f /var/local/xwin_enable_xwayland ] ; then
 	XWAYLAND=1
+elif [ -e /usr/bin/startxwayland -a ! -e /usr/bin/Xorg ]; then
+	XWAYLAND=1
+	touch /var/local/xwin_enable_xwayland
```

Comment:
If the user runs `apt install xserver-xorg`, we want to show the "switch to X.Org" button